packages/frontend/src/components/dialogs/ProxyConfiguration/index.tsx

```diff
@@ -370,7 +370,7 @@ export default function ProxyConfiguration(
       <DialogFooter>
         {showNewProxyForm && (
           <form>
-            <DeltaInput
+            <DeltaTextarea
```

Comment:
Now one cannot submit the form with "Enter". I would say that keeping this an `<input>` is better. One does not usually need the proxy line.

Reply:
I would prefer to make the whole URL visible by wrapping it into the text area which is not achievable in a text input afaik